NEWS.md

```diff
@@ -3,6 +3,7 @@ ClimaLand.jl Release Notes
 
 main
 -------
+- ![breaking change][badge-💥breaking] Add moisture stress module to the canopy model PR[#1387](https://github.com/CliMA/ClimaLand.jl/pull/1387)
```

Comment:
Could you add a little more detail about why this is a breaking change and what users should do to update for the change?